colossalai/fx/__init__.py

```diff
@@ -1,3 +1,10 @@
-from .tracer import ColoTracer, meta_trace
+try:
+    from . import _meta_registrations
+    META_COMPATIBILITY = True
+except:
+    import torch
+    META_COMPATIBILITY = False

```

Comment:
only import these once
`META_COMPATIBILITY` is used for function `check_meta_compatibility()` in `_compatibility.py`